openupgrade_scripts/base/14.0.1.3/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,123 @@
+---Models in module 'base'---
```

Comment:
Now that we're in the works of refactoring and all that stuff, I have to tell you something: I hate the syntax of these files.

Couldn't we just have a YAML, which supports proper comments and syntax highlighting, and besides allows automated parsing to check migration completion status? Or even a CSV... but not this please :see_no_evil: 

(Note: of course just a personal preference and absolutely not important. But really, this is awful :laughing:)

Reply:
Hi @yajo. Well it's a distinct point. You're probably right. But it requires a big extra work. Feel free to make a PR against upgrade_analysis module, that is responsible of the generation of the files.